npm-shrinkwrap.json

```diff
@@ -2272,7 +2272,7 @@
     },
     "browserify-aes": {
       "version": "1.2.0",
-      "resolved": "https://registry.npmjs.org/browserify-aes/-/browserify-aes-1.2.0.tgz",
```

Comment:
Were these changes to `npm-shrinkwrap.json` intentional? Most of them point us to the npm registry via HTTP instead of HTTPS. I think we should remove them from the PR.

Reply:
Thanks, @jakobsandberg. I rolled them back as you suggested.